samples/compiler_plugins/xnnpack/test/batch-matrix-multiply-e2e.mlir

```diff
@@ -1,4 +1,6 @@
-// RUN: iree-compile --iree-hal-target-backends=llvm-cpu --iree-plugin=xnnpack %s | \
+// RUN: iree-compile --iree-hal-target-backends=llvm-cpu --iree-plugin=xnnpack --compile-to=flow %s | \
+// RUN: iree-opt --inline | \
```

Comment:
As mentioned, it is fine to modify the precodegen with our code with a comment, so that later we can find it easily. You can do it later though.